mscore/metaedit.cpp

```diff
@@ -60,6 +61,15 @@ MetaEditDialog::MetaEditDialog(Score* s, QWidget* parent)
             grid->addWidget(text, idx, 1);
             ++idx;
             }
+      revealButton->setIcon(*icons[int(Icons::fileOpen_ICON)]);
+#if defined(Q_OS_WIN)
+      revealButton->setToolTip("Show in Explorer");
```

Comment:
not translatable, make it `revealButton->setToolTip(tr("Show in Explorer"));`. Same for the next similar ones

Reply:
Not sure though why we'd need 3 different strings for 3 different platforms, just `revealButton->setToolTip(tr("Open file location"));` for all platforms should be enough

Reply:
Yes, I hesitated too, but it looks like there's a standard for mac and Windows. Plus, apart from 2 more strings to translate, it doesn't do any harm. :)

Reply:
[QtCreator's handling of it](https://github.com/qt-creator/qt-creator/blob/master/src/plugins/coreplugin/fileutils.cpp#L129)

Reply:
But would make the handbook unneccessarily complex too

Reply:
BTW, using those `QApplication::translate("Core::Internal", ...) would mean we don't need to translate at all but take Qt's translations. Not sure whether we want that though

Reply:
Doesn't work for me.
![image](https://user-images.githubusercontent.com/35939574/73236177-6f1b9a80-415f-11ea-8519-d595493833a9.png)

Reply:
Maybe that string is not transtaled by Qt for ypour language? But as said, I'm not even sure we want that

Reply:
Or maybe we don't have access to `"Core::Internal"`

Reply:
Core::Internal is from Qt Creator, I dont think we have access to it